src/private/remoteCamera.ts

```diff
@@ -0,0 +1,359 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError } from '../public/interfaces';
+export namespace remoteCamera {
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent patricipant details needed to request control of camera.
+   */
+  export interface ParticipantInput {
+    /**
+     * Id of participant.
+     */
+    id: string;
+    /**
+     * Display name of participant.
+     */
+    displayName?: string;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent a patricipant in a meeting.
+   */
+  export interface Participant extends ParticipantInput {
+    isCapable: boolean;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate possible camera control commands.
+   */
+  export enum ControlCommand {
+    Reset = 'Reset',
+    ZoomIn = 'ZoomIn',
+    ZoomOut = 'ZoomOut',
+    PanLeft = 'PanLeft',
+    PanRight = 'PanRight',
+    TiltUp = 'TiltUp',
+    TiltDown = 'TiltDown',
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to hold the array of participants with controllable-cameras in a meeting.
+   */
+  export interface CapableParticipants {
+    participants: Participant[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to return error reason when a handler error occurs.
+   */
+  export interface HandlerFailed {
+    handlerError: ErrorReason;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to track the controlled device's state.
+   */
+  export interface DeviceStateChanged {
```

Comment:
can we directly return DeviceState. No need to have class for one field.

Reply:
This is type is taken from the CDL / schema code; I was remaining consistent with the data returned by those queries/mutations/subscriptions